connect-examples/oauth/php/README.md

```diff
@@ -12,7 +12,7 @@ along with the comments included in `sandbox_callback.php`.
 
 ### Requirements
 
-* 7.1 <= PHP < 8.1
+* 7.1 <= PHP
```

Comment:
We're still supposed to be under 8.2, as shown here: https://github.com/square/square-php-sdk/blob/master/composer.json#L21

- [ ] Set to 8.2

Reply:
Isn't 8.1 the latest available: https://www.php.net/supported-versions.php ?

Reply:
Oh interesting. I had no idea. It's set in composer but I don't feel super strongly bout enforcing it here since it's enforced in the SDK